flexiblepricing/mail_api.py

```diff
@@ -0,0 +1,89 @@
+"""
+Utils for mail
+"""
+import logging
+
+from django.core.exceptions import ValidationError
+from mitol.mail.api import get_message_sender
+
+from courses.models import CourseRun
+from flexiblepricing.constants import (
+    FLEXIBLE_PRICE_APPROVAL_MESSAGE,
+    FLEXIBLE_PRICE_APPROVAL_SUBJECT,
+    FLEXIBLE_PRICE_DOCUMENTS_RECEIVED_MESSAGE,
+    FLEXIBLE_PRICE_DOCUMENTS_RESET_MESSAGE,
+    FLEXIBLE_PRICE_RESET_SUBJECT,
+    FLEXIBLE_PRICE_DOCUMENTS_RECEIVED_SUBJECT,
+    FLEXIBLE_PRICE_EMAIL_BODY,
```

Comment:
nit: We can adjust the imports in a more readable way e.g. importing `FLEXIBLE_PRICE_DOCUMENTS_*` in order and the same for the others.

Reply:
FYI I renamed and regrouped these so that they're a bit cleaner but ultimately I'm leaving this mostly alone - I'd prefer we get an actual email template (like we have for receipts/etc.) and use that instead, which would eliminate these constants eventually.